portfolio/src/main/java/com/google/sps/servlets/RandomGameServlet.java

```diff
@@ -15,18 +15,34 @@
 package com.google.sps.servlets;
 
 import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
-@WebServlet("/data")
-public class DataServlet extends HttpServlet {
+@WebServlet("/random-game")
+public final class RandomGameServlet extends HttpServlet {
+
+    private List<String> games;
```

Comment:
fix the indent here (maybe look at autoformatters for the future?)

Reply:
I tried finding an autoformatter for the cloud shell, but I don't think it's supported in the editor. but good catch! realised that the original code used 2 spaces for indenting. will stick to this in the future.